sdks/java/extensions/euphoria/src/main/java/org/apache/beam/sdk/extensions/euphoria/core/client/operator/Distinct.java

```diff
@@ -111,70 +111,74 @@ public static OfBuilder named(@Nullable String name) {
      * <p>This is, while windowing will be applied on basis of original input elements, the distinct
      * operator will be carried out on the transformed elements.
      *
-     * @param <T> the type of the transformed elements
+     * @param <KeyT> the type of the transformed elements
      * @param mapper a transform function applied to input element
      * @return the next builder to complete the setup of the {@link Distinct} operator
      */
-    default <T> WindowByBuilder<T> mapped(UnaryFunction<InputT, T> mapper) {
+    default <KeyT> WindowByBuilder<InputT, KeyT> mapped(UnaryFunction<InputT, KeyT> mapper) {
       return mapped(mapper, null);
     }
 
-    <T> WindowByBuilder<T> mapped(
-        UnaryFunction<InputT, T> mapper, @Nullable TypeDescriptor<T> outputType);
+    <KeyT> WindowByBuilder<InputT, KeyT> mapped(
```

Comment:
Reason why parameter was named `T` is that `KeyT` hides class level `KeyT` parameter